ResearchUXFactory/SBAURLLearnMoreAction.swift

```diff
@@ -73,6 +73,10 @@ public final class SBAURLLearnMoreAction: SBALearnMoreAction {
         vc.url = learnMoreURL
         vc.html = learnMoreHTML
         vc.navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .done, target: vc, action: #selector(vc.dismissViewController))
+        let doneButtonColor = UIColor.color(for: "learnMoreDoneButtonColor")
```

Comment:
Why "learnMoreDoneButtonColor"?

Also, use UIColor+Utilities so that apps can override the extension with a different color ala Zeplin definitions.